apps/setlang

```diff
@@ -13,10 +13,10 @@ thisdir="$(dirname $0)"
 if [ -z $LOG ]; then
 	. $thisdir/../config/GLOBALS
 fi
-echo -ne "\n$ME-$thisname---->\n" | $TEE -i -a $LOG
+echo -ne "$ME-$thisname---->\n" | $TEE -i -a $LOG
```

Comment:
the extra line break was on purpose. If there are too many, check GLOBALS

Reply:
you do not need it here as linefeed is done on the GOLBALS side